tools/flaky-test-reporter/jsonreport/jsonreport.go

```diff
@@ -110,6 +111,23 @@ func GetReportForRepo(repo string, buildID int) (*Report, error) {
 	return report, nil
 }
 
+//GetReposWithReports gets all the repositories that are currently being reported on
+func GetReposWithReports() ([]string, error) {
```

Comment:
this logic should be combined with `getJSONForBuild` above, since they should be looking at the same build